### DISCUSSION TITLE
My some question about different method for INS flow in Finite Element Methods

### DISCUSSION THREAD

**Discussion by suqingji** [idaholab/moose#25212] [Q&A Modules: Navier-Stokes]
Dear all developers,
There are some different seperate method to solve `INS` problem with finite element method in `MOOSE`, such as being used in `lid_driven_split` and `lid_driven_chorin`. So Do they have any advatages and disadvatages. Could you please introduce about them and show some articles about those alogrithm.

Best wishes.

**Comment by GiudGiud** [comment]
Hello

@freiler can comment on the Chorin projection
there should be a bibliography file in the navier stokes module doc / bib folder if you want to check the publications we have / we reference in the docs

Guillaume

**Reply by suqingji** [reply]
Thanks very much.

**Reply by freiler** [reply]
The Chorin method involves splitting the Navier-Stokes equations into two separate steps: a prediction step for the velocity field and a correction step for the pressure field. Here's how it works in a nutshell:

1) Prediction Step: In this step, a tentative velocity field is computed by solving the advection and diffusion terms of the Navier-Stokes equations without considering the current pressure gradient term and linearizing the advective term. This step essentially predicts the velocity field at the next time step.

2) Pressure Correction Step: The predicted velocity field is then used to compute a provisional pressure field using a Poisson equation derived from the incompressibility constraint. The pressure correction is applied to remove any divergence from the predicted velocity field.

3) Correction Step: The predicted velocity field is corrected by subtracting the gradient of the pressure field computed in the previous step. This ensures that the final velocity field is divergence-free.

Some Advantages:

-Simplicity: Chorin's method is one of the easiest concepts of segregated solvers to implement and understand.
-Speed: Compared to fully coupled solvers.

Disadvantages:

-Accuracy: The accuracy can be limited compared to more advanced methods like fractional step methods or implicit methods.
-Pressure Oscillations: Chorin's method can sometimes lead to spurious pressure oscillations, especially in situations with strong gradients in the flow field.

Reference about Chorin
1 Chorin, A.J.: Numerical solution of the Navier-Stokes equations. Math.Comp. 22, 745–762 (1968).
